documentation/staging/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -40,32 +38,29 @@ This is because a Model in Image domain home is regenerated from the model on ev
 There are two approaches for propagating model updates to a running Model in Image domain
 without first shutting down the domain:
 
+- _Offline updates_: [Offline updates](#offline-updates) are propagated to WebLogic pods by updating your model
+  and then initiating a domain roll, which generates a new domain configuration,
+  restarts the domain's WebLogic Server Administration Server with the updated configuration,
+  and then restarts the other pods in the cluster.
```

Comment:
I see that this was the existing text, but saying, "and then restarts the other pods in the cluster." is confusing as the Administration Server is not in a cluster and this also implies that there is exactly one cluster and that all other servers are clustered servers. This is true for many of our samples, but isn't universally true. Can this say something like, "... and then restarts other running servers."? I don't think that we need to even mention here that clustered servers will be restarted in a roll as to preserve cluster availability.

Reply:
@rjeberhard Not sure exactly which sentence should be removed or modified according to this feedback, "I don't think that we need to even mention here that clustered servers will be restarted in a roll as to preserve cluster availability." 

Do you want this sentence changed to, "[Offline updates](#offline-updates) are propagated to WebLogic pods by updating your model and then restarts other running servers."

Reply:
You have the right sentence, but I was commenting on the content on line 44 that presently reads, "and then restarts the other pods in the cluster." I was pointing out that the Administration Server isn't in a cluster and so it doesn't make sense to say the "other pods in the cluster" unless the sentence is implying that there is always exactly one cluster and that all other servers are members of that one cluster -- that is true for many of our samples, but isn't generally true.

Reply:
Yes, that part I got. Thank you.